compose/ui/ui/src/webMain/kotlin/androidx/compose/ui/window/ComposeWindow.web.kt

```diff
@@ -37,21 +45,97 @@ fun ComposeViewport(
     configure: ComposeViewportConfiguration.() -> Unit = {},
     content: @Composable () -> Unit = { }
 ) {
-    InternalComposeViewport(
-        viewportContainerId = viewportContainerId,
-        configure = configure,
-        content = content
-    )
+    onDomReady {
+        val providedContainer = if (viewportContainerId != null) {
+            document.getElementById(viewportContainerId) ?: error("failed to find element by viewportContainerId: '$viewportContainerId'")
+        } else {
+            document.body ?: error("failed to find <body> element")
+        }
+
+        ComposeViewport(providedContainer, configure, content)
+    }
 }
 
-internal expect fun InternalComposeViewport(
-    viewportContainerId: String? = null,
+/**
+ * EXPERIMENTAL! Might be deleted or changed in the future!
```

Comment:
We probably need to stabilize it in 1.11 (now?) because now it seems like

<img width="1080" height="759" alt="Image" src="https://github.com/user-attachments/assets/94a1acb3-56ed-4b6f-8f06-1c2c79631185" />